### DISCUSSION TITLE
Does dbus-mqtt-devices work with VenusOS 3.30+?

### DISCUSSION THREAD

**Discussion by jonofe** [freakent/dbus-mqtt-devices#114] [Q&A]
Hi guys,
just tried to update from VenusOS 3.22 to the new stable 3.30.
None of my dbus-mqtt.devices (1 x grid meter, 2 x pvinverter) do show up after update.
Is that a known incompatibility or do I have to change something in my scripts?
Best regards
André

**Comment by jonofe** [comment]
After updating from c0.6.4 to 0.6.6, I got it to work.
Thanks for that great project.
Regards
André

**Reply by freakent** [reply]
Glad you got it working. I'm getting ready to make a new 0.8 release with some exciting new features 😀
